static/app/views/issueDetails/traceTimeline/traceIssue.tsx

```diff
@@ -92,19 +92,15 @@ export function getTitleSubtitleMessage(event: TimelineEvent) {
       // It uses metadata.value which could differ depending on what error.value is used in the event manager
       // TODO: Add support for chained exceptions since we grab the value from the first stack trace
       // https://github.com/getsentry/sentry/blob/a221f399d2b4190f2631fcca311bdb5b3748838b/src/sentry/eventtypes/error.py#L115-L134
-      message = event['error.value'].at(-1);
+      message = event['error.value'].at(-1) || '';
```

Comment:
Otherwise, we get this error: `Type 'string | undefined' is not assignable to type 'string'.`

Reply:
nit: it has the same behavior here, but we generally want to use `??` unless you really need `||`